tasks/test_results_processor.py

```diff
@@ -76,9 +76,55 @@ def get_repo_flags(
     return repo_flag_mapping
 
 
+@dataclass
+class PytestName:
+    actual_class_name: str
+    test_file_path: str
+
+
+@dataclass
+class argProcessingResult:
+    network_files: list[str] | None
+    parsing_results: list[ParsingInfo]
+
+
 class TestResultsProcessorTask(BaseCodecovTask, name=test_results_processor_task_name):
     __test__ = False
 
+    def compute_name(
+        self,
+        framework: Framework,
+        raw_classname: str,
+        raw_name: str,
+        filename: str | None,
+        network: list[str] | None,
+    ) -> str:
+        match framework:
+            case Framework.Jest:
+                name = raw_name
+                return name
```

Comment:
simplification suggestion:
```suggestion
                return raw_name
```